go.mod

```diff
@@ -4,13 +4,14 @@ go 1.13
 
 require (
 	cloud.google.com/go/storage v1.5.0
-	github.com/ghodss/yaml v0.0.0-20150909031657-73d445a93680
-	github.com/google/go-github/v29 v29.0.2
+	github.com/ghodss/yaml v1.0.0
+	github.com/google/go-github/v29 v29.0.3
 	github.com/google/martian v2.1.0+incompatible
-	github.com/spf13/cobra v0.0.5
+	github.com/spf13/cobra v0.0.6
 	github.com/spf13/pflag v1.0.5
-	golang.org/x/oauth2 v0.0.0-20191202225959-858c2ad4c8b6
+	golang.org/x/oauth2 v0.0.0-20200107190931-bf48bf16ab8d
 	google.golang.org/api v0.15.0
 	k8s.io/apimachinery v0.17.2
-	sigs.k8s.io/yaml v1.1.0
+	k8s.io/release v0.2.5 // indirect
```

Comment:
🤔 what has pulled this in?

Reply:
We don't have any kind of CI here so `go mod tidy` may sort it out? otherwise `go mod why`? 😄